src/I18n/DateFormatTrait.php

```diff
@@ -276,8 +292,11 @@ public static function setToStringFormat($format): void
      * will be used for formatting the date part of the object and the second position
      * will be used to format the time part.
      *
+     * Alternatively, the format can provide a callback. In this case, the callback
```

Comment:
This should be moved to I18nDateTimeInterface then @inheritDoc instead.

Reply:
Going to remove this as the property is defined on each class that implements this trait.